docs/planning/timeline-editing.mdx

```diff
@@ -40,40 +40,94 @@ Click on the plus button to add a new row to the timeline. A row will be added b
+To visualize activities on the timeline add a new layer using the plus button in the Layers section. By default new layers have a layer type of Activity. Additionally, the default Aerie UI view creates a row with a single activity layer with all activities grouped by type. Manage which activities are visualized in an activity layer by clicking on the search input for the layer and selecting activities. Activities can be filtered by substring match using the input. Once you have a filter applied you can also select all matches using the Select X activities button at the bottom of the result list. You can also manage the color of the layer using the color picker. Below is a demonstration of this workflow.
 
 <video controls>
   <source src={timelineActivityEditingDemo} />
 </video>
 
-### Managing Activity Layer Height
-
-Rows by default have a manually specified height. The UI will pack directives and simulated activities in a layer to make the most efficient use of the available space. However if the packed content is still too big for the manually specified height the content will begin to overdraw upon itself. To resolve this issue you can input a larger manual height for the row or specify that the row automatically calculate the minimum height needed to prevent overdraw. Control the row height manually from within the Details section of the timeline row editor or by dragging on the bottom of the row. Alternatively use the Auto option from within the Resize Mode dropdown in the Details section to have the row automatically resize to the content.
+<br />
+<br />
 
-:::info Note
-Use of Auto mode can result in degraded Aerie UI performance when visualizing thousands of activities on the timeline. It is recommended that you use a smaller, manually specified height for this scenario. Alternatively you can filter the activities down to a more manageable set to improve performance.
+:::info
+Note All activity layers within a row are drawn on the same canvas so that all of the activities are grouped and packed together correctly. Activities will take on the color of the layer that they came from.
 :::
 
-Below is a demonstration of this workflow.
+### Manage how Activities are Rendered
 
-<video controls>
-  <source src={timelineActivityHeightEditingDemo} />
-</video>
+An Activity Options section will appear in the row editor when the row has at least one activity layer. Activity options govern how activities are rendered for this row.
+
+<figure>
+  <img
+    alt="Aerie UI - Timeline Activity Options"
+    src={timelineActivityOptionsEditor}
+    style={{ maxHeight: '100%', width: '100%' }}
+  />
+  <figcaption>Figure 3: Aerie UI Timeline Activity Options Editor - Manage Activity Rendering</figcaption>
+</figure>
+
+#### 1. Activity Display Mode
+
+Activities can be rendered in two ways:
```

Comment:
Minor, nonblocking: would it be possible to add a side by side of the same timeline using these two modes?

Reply:
Great idea